### DISCUSSION TITLE
[feature reguest] Filter option on number of hashtags in post.

### DISCUSSION THREAD

**Discussion by 8Kuula** [mastodon/mastodon#31537] [General]
If post has more than X number hashtags it would get filtered.

Looking way to filter out some plain spam posts from my home feed, and i think this feature could help. Looks of it spam posts have many hashtags just to show up everywhere.

**Comment by quicoto** [comment]
Aren't the moderation tools to mute/block entire domains/instances not good enough for fighting spam?
